example/src/CameraPage.tsx

```diff
@@ -200,7 +200,10 @@ export function CameraPage({ navigation }: Props): React.ReactElement {
   const frameProcessor = useFrameProcessor((frame) => {
     'worklet';
     const values = examplePlugin(frame);
-    console.log(`Return Values: ${JSON.stringify(values)}`);
+    const valuesSwift = examplePluginSwift(frame);
+
+    console.log(`Return OBJC Values: ${JSON.stringify(values)}`);
+    console.log(`Return Swift Values: ${JSON.stringify(valuesSwift)}`);
```

Comment:
Please remove all changes in this file

Reply:
Reverted in mean time as your project.

Reply:
No danger, just slower & not really needed to run everything in there imo

Reply:
does help catch future regressions though :) but I see your point

Reply:
True. But what happens on Android? There's no `examplePluginSwift`.